no_drama/__main__.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+
+import argparse
+import tempfile
+import os
+import os.path
+import shutil
+import subprocess
+import zipfile
+import glob
+import fnmatch
+
+from string import Template
+from functools import partial
+
+
+this_file = os.path.realpath(__file__)
+this_directory = os.path.dirname(this_file)
+templates_dir = os.path.join(this_directory, 'templates')
+build_lib = os.path.join(this_directory, 'build_lib')
+build_files = os.path.join(this_directory, 'build_files')
+
+
+def save_wheels(destination, packages=None, requirements_file=None,
+                find_links=None):
+    call_list = ["pip", "wheel", "--wheel-dir=%s" % destination]
+
+    if packages is not None:
+        call_list += packages
+
+    if find_links is not None:
+        call_list += ['--find-links=%s' % find_links]
+
+    if requirements_file is not None:
```

Comment:
Open an issue in https://github.com/cfpb/back-end! :grin:

[Google's Python Style Guide](https://google.github.io/styleguide/pyguide.html#True/False_evaluations) also mentions this as an exception to trusting implicit falseness, for reasons of ambiguity.

:grin: